muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/workspace/workspaceJournal/dialogs/delete-journal-comment.tsx

```diff
@@ -0,0 +1,153 @@
+import "~/sass/elements/link.scss";
+import "~/sass/elements/form.scss";
+import "~/sass/elements/buttons.scss";
+import * as React from "react";
+import { connect, Dispatch } from "react-redux";
+import { bindActionCreators } from "redux";
+import { AnyActionType } from "~/actions";
+import { i18nType } from "~/reducers/base/i18n";
+import Dialog from "~/components/general/dialog";
+import Button from "~/components/general/button";
+import { StateType } from "~/reducers";
+import { JournalComment } from "~/@types/journal";
+import {
+  DeleteWorkspaceJournalCommentTriggerType,
+  deleteWorkspaceJournalComment,
+} from "../../../../actions/workspaces/journals";
```

Comment:
Is there some particular reason not to use `~` instead of `../../ `etc.with the paths when importing?

There might be more of these imports which differs from the rest.